go.mod

```diff
@@ -28,6 +28,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/dylanmei/winrmtest v0.0.0-20210303004826-fbc9ae56efb6
 	github.com/go-test/deep v1.0.3
+	github.com/go-viper/mapstructure/v2 v2.0.0-alpha.1
```

Comment:
We should not bring in alpha here, we should use a stable release.

Reply:
Correction: this is fine: https://github.com/go-viper/mapstructure/compare/v1.6.0...v2.0.0-alpha.1

2.0.0-alpha is just a rename of the module from 1.6.0 with